### PR TITLE
Fix init for Windows 10

### PR DESCRIPTION
**Pull request by tklengyel**
The function `init_from_rekall_profile_real` has been very messy. There are 4 methods in it to find the KPCR for Windows, but any error in any of them results in bailing out from initializing instead of trying any of the other methods still available. I split out the 4 methods into separate functions so that each can be tried in succession.

### CODE REVIEW

**Comment by smaresca** [comment]
+1000

File: libvmi/os/windows/core.c [hunk 1]
```diff
@@ -746,129 +746,161 @@ find_windows_version_from_rekall_profile(vmi_instance_t vmi)
...
+            dbprint(VMI_DEBUG_MISC, "** driver_get_vcpureg(..) failed.\n");
+            goto done;
+        }
+
+        if ( VMI_SUCCESS == kpcr_find1(vmi, windows, kpcr_reg) ) {}
+        else if ( VMI_SUCCESS == kpcr_find2(vmi, windows) ) {}
+        else if ( VMI_SUCCESS == kpcr_find3(vmi, windows) ) {}
+        else if ( VMI_SUCCESS == kpcr_find4(vmi, windows, kpcr_reg) ) {}
+        else goto done;
+
```
Comment (smaresca) [inline libvmi/os/windows/core.c]: the above is a bit odd, though I see why you did it

Comment (tklengyel) [reply libvmi/os/windows/core.c]: Yea, it's a bit odd from a C perspective but I find it quite readable `¯\_(ツ)_/¯`

Comment (tklengyel) [reply libvmi/os/windows/core.c]: The other option I was contemplating is adding a `kpcr_found` goto label, but I think this is nicer.

**Pull request merged**